swarm/core.py

```diff
@@ -39,17 +41,18 @@ def condense(result: List[Any]):
         return np.array(result)
 
 
-@util.time_me
-def swarm_train(bee_trainer, num_swarm=50, seed=None, fields=None):
+def swarm_train(bee_trainer, bee_params=None, num_bees=50, seed=None, fields=None) -> Dict[str, Any]:
```

Comment:
Btw, this changes the way we do `swarm_training`,  basically making `make_bee` unecessary since you can pass the params directly into `swarm_train`. 

This is backwards incompatible, but should be a fairly simple fix, i.e. 
`swarm_train(bee, 50, 10, "ypred,loss")` becomes `swarm_train(bee, None, 50, 10, "ypred,loss")`

Reply:
Ok, that seems doable enough... I'd be happy trying to implement those fixes to make it work.

Reply:
Since you went first, I did check your code invocations and it should be fine.